docs/source/guide/index.rst

```diff
@@ -913,6 +913,28 @@ This command adds the provided artifact to the TUF Metadata using the RSTUF REST
     │ --help          -h    Show this message and exit.                                                │
     ╰──────────────────────────────────────────────────────────────────────────────────────────────────╯
 
+.. rstuf-cli-artifact-download
+
+Artifact Download (``download``)
+
+This command allows downloading an artifact from a provided repository using the RSTUF REST API.
+
+.. code::
+
+    > rstuf artifact download --help
+
+    Usage: rstuf artifact download [OPTIONS] ARTIFACT_NAME
+
+    Downloads artifacts to the TUF metadata.
```

Comment:
Please add a line here that this artifact will be downloaded from a setting named `current_repository`
and with one sentence mention how you can change that repository.
I just want to make sure that a user of `rstuf artifact download` understands both where he is downloading from and how to change that.

Reply:
```suggestion
    Downloads artifact(s) to the TUF metadata.
```

Reply:
`current_repostitory` is only referred to when an rstuf config is used. May be you mean adding few lines about how to add `artifacts` configuration.